pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[tool.poetry]
+name = "kintree"
+version = "0.3.6"
+description = "Fast part creation in KiCad and InvenTree"
+authors = ["Placeholder Name <placeholder@test.com>"]
+license = "GPL3"
+
+[tool.poetry.dependencies]
+
+python = ">=3.7,<3.10"
+altgraph = ">=0.17,<1.0"
+certauth = ">=1.3.0,<2.0"
+certifi = ">=2020.6.20,<2021.0"
+cffi = ">=1.14.1,<2.0"
+chardet = ">=3.0.4,<4.0"
+coverage = ">=5.2.1,<6.0"
+coveralls = ">=2.1.2,<3.0"
+cryptography = ">=3.0,<4.0"
+digikey-api = ">=0.4.0,<1.0"
+fake-useragent = ">=0.1.11,<1.0"
+fuzzywuzzy = ">=0.18.0,<1.0"
+idna = ">=2.10,<3.0"
+inflection = ">=0.5.0,<1.0"
+inventree = ">=0.2.4,<1.0"
+pycparser = ">=2.20,<3.0"
+pyinstaller = ">=4.0,<5.0"
```

Comment:
Technically pyinstaller-* are dev dependencies.

Reply:
@eeintech This, too.

Reply:
So do you mean removing this dependency and installing it only when the `inv build` is ran? I would like to keep the `inv build` available for all users.